apis/node/health.yaml

```
@@ -17,7 +17,7 @@ get:
     "200":
       description: Node is ready
     "206":
-      description: Node is syncing but can serve incomplete data
+      description: Either the consensus node is syncing, or its execution node is optimistic, so data served may be incorrect
```

The spec does not use the term "consensus node" while "beacon node" is used 85 times

```suggestion
      description: Either the beacon node is syncing, or its execution node is optimistic, so data served may be incorrect
```

Probably personal preference but in my mind beacon node is also more accurate, a consensus node might suggest that it also includes a validator client as a beacon node by itself can't participate in consensus